controllers.js

```diff
@@ -73,6 +74,31 @@ exports.fetchComments = (req, res, next) => {
     .catch(next)
 };
 
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  if (!username || !body) {
+    return res.status(400).send({ message: 'Bad Request! Missing required fields.' });
+  }
+
+  if (isNaN(article_id)) {
+    return res.status(400).send({ message: 'Unable to post! Invalid article ID.' });
+  }
```

Comment:
You can remove these checks since the model will handle them for you. There's no need to verify if the article exists either.